cache/fast_lru_cache.cc

```diff
@@ -182,7 +167,7 @@ inline int LRUHandleTable::FindSlot(const Slice& key,
       return -1;
     }
     h->displacements += displacement;
-    current = BinaryMod(current + increment, length_bits_);
+    current = BinaryMod<uint32_t>(current + increment, length_bits_);
```

Comment:
Nit: explicit template parameter unnecessary when it can be inferred from function arguments